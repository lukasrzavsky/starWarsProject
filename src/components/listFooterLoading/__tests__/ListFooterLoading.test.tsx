import React from "react";
import renderer from "react-test-renderer";
import { ListFooterLoading } from "..";

describe("<ListFooterLoading />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ListFooterLoading />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("has 1 child", () => {
    const tree = renderer
      .create(<ListFooterLoading />)
      .toJSON() as renderer.ReactTestRendererJSON;
    expect(tree.children.length).toBe(1);
  });
});
