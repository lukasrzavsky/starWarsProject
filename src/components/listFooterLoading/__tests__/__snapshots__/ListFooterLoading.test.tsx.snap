// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListFooterLoading /> renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#000",
      "height": 40,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <ActivityIndicator />
</View>
`;
